From fa5e7f23e2efe54b0c19719bea01c20a5c46f361 Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@freescale.com>
Date: Sun, 5 Feb 2012 12:27:11 -0600
Subject: [PATCH 232/498] ENGR00173826: Fix the SGTL5000 strings as per
 linux-2.6.38

Fix the SGTL5000 strings as per linux-2.6.38 kernel as
ICS is running with linux-2.6.38 or further kernels only.

Signed-off-by: Nitin Garg <nitin.garg@freescale.com>
---
 alsa/alsa_imx.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/alsa/alsa_imx.cpp b/alsa/alsa_imx.cpp
index 5183a22..631b40c 100755
--- a/alsa/alsa_imx.cpp
+++ b/alsa/alsa_imx.cpp
@@ -46,12 +46,12 @@
 #define AUDIOCARD_DEVICE_CS42888 "HiFi CS42888-0"
 #else
 
-#define AUDIOCARD_DEVICE_SGTL5000_HIFI "SGTL5000 SGTL5000-0"
+#define AUDIOCARD_DEVICE_SGTL5000_HIFI "HiFi sgtl5000-0"
 #define AUDIOCARD_DEVICE_WM8958_HIFI "WM8994 HiFi WM8994 AIF1-0"
 #define AUDIOCARD_DEVICE_WM8958_VOICE "WM8994 Voice WM8994 AIF2-1"
 #define AUDIOCARD_DEVICE_WM8958_BT "WM8994 BT WM8994 AIF3-2"
 #define AUDIOCARD_DEVICE_HDMI "IMX HDMI TX mxc-hdmi-soc-0"
-#define AUDIOCARD_DEVICE_SPDIF "IMX SPDIF mxc spdif-0"
+#define AUDIOCARD_DEVICE_SPDIF "IMX SPDIF mxc-spdif-0"
 #define AUDIOCARD_DEVICE_CS42888 "HiFi CS42888-0"
 #endif
 
-- 
1.8.0

